runtime/oti/j9java8packages.gperf

```diff
@@ -0,0 +1,2068 @@
+com/sun/corba/se/internal/Interceptors
+com/sun/corba/se/internal/POA
+com/sun/corba/se/internal/corba
+com/sun/corba/se/internal/iiop
+com/sun/corba/se/org/omg/CORBA
+com/sun/corba/se/pept/broker
+com/sun/corba/se/pept/encoding
+com/sun/corba/se/pept/protocol
+com/sun/corba/se/pept/transport
+com/sun/corba/se/spi/activation
+com/sun/corba/se/spi/activation/InitialNameServicePackage
+com/sun/corba/se/spi/activation/LocatorPackage
+com/sun/corba/se/spi/activation/RepositoryPackage
+com/sun/corba/se/spi/copyobject
+com/sun/corba/se/spi/encoding
+com/sun/corba/se/spi/extension
+com/sun/corba/se/spi/ior
+com/sun/corba/se/spi/ior/iiop
+com/sun/corba/se/spi/legacy/connection
+com/sun/corba/se/spi/legacy/interceptor
+com/sun/corba/se/spi/logging
+com/sun/corba/se/spi/monitoring
+com/sun/corba/se/spi/oa
+com/sun/corba/se/spi/orb
+com/sun/corba/se/spi/orbutil/closure
+com/sun/corba/se/spi/orbutil/fsm
+com/sun/corba/se/spi/orbutil/proxy
+com/sun/corba/se/spi/orbutil/threadpool
+com/sun/corba/se/spi/presentation/rmi
+com/sun/corba/se/spi/protocol
+com/sun/corba/se/spi/resolver
+com/sun/corba/se/spi/servicecontext
+com/sun/corba/se/spi/transport
+com/sun/demo/jvmti/hprof
+com/sun/image/codec/jpeg
+com/sun/imageio/plugins/bmp
+com/sun/imageio/plugins/common
+com/sun/imageio/plugins/gif
+com/sun/imageio/plugins/jpeg
+com/sun/imageio/plugins/png
+com/sun/imageio/plugins/wbmp
+com/sun/imageio/spi
+com/sun/imageio/stream
+com/sun/istack/internal
+com/sun/istack/internal/localization
+com/sun/istack/internal/logging
+com/sun/java/browser/dom
+com/sun/java/browser/net
+com/sun/java/swing
+com/sun/java/swing/plaf/gtk
+com/sun/java/swing/plaf/gtk/resources
+com/sun/java/swing/plaf/motif
+com/sun/java/swing/plaf/motif/resources
+com/sun/java/swing/plaf/nimbus
+com/sun/java/swing/plaf/windows
+com/sun/java/swing/plaf/windows/resources
+com/sun/java/util/jar/pack
+com/sun/jmx/defaults
+com/sun/jmx/interceptor
+com/sun/jmx/mbeanserver
+com/sun/jmx/remote/internal
+com/sun/jmx/remote/protocol/iiop
+com/sun/jmx/remote/protocol/rmi
+com/sun/jmx/remote/security
+com/sun/jmx/remote/util
+com/sun/jmx/snmp
+com/sun/jmx/snmp/IPAcl
+com/sun/jmx/snmp/agent
+com/sun/jmx/snmp/daemon
+com/sun/jmx/snmp/defaults
+com/sun/jmx/snmp/internal
+com/sun/jmx/snmp/mpm
+com/sun/jmx/snmp/tasks
+com/sun/jndi/cosnaming
+com/sun/jndi/dns
+com/sun/jndi/ldap
+com/sun/jndi/ldap/ext
+com/sun/jndi/ldap/pool
+com/sun/jndi/ldap/sasl
+com/sun/jndi/rmi/registry
+com/sun/jndi/toolkit/corba
+com/sun/jndi/toolkit/ctx
+com/sun/jndi/toolkit/dir
+com/sun/jndi/toolkit/url
+com/sun/jndi/url/corbaname
+com/sun/jndi/url/dns
+com/sun/jndi/url/iiop
+com/sun/jndi/url/iiopname
+com/sun/jndi/url/ldap
+com/sun/jndi/url/ldaps
+com/sun/jndi/url/rmi
+com/sun/management
+com/sun/management/jmx
+com/sun/media/sound
+com/sun/naming/internal
+com/sun/net/httpserver
+com/sun/net/httpserver/spi
+com/sun/net/ssl/internal/www/protocol/https
+com/sun/nio/file
+com/sun/nio/sctp
+com/sun/org/apache/xml/internal/security
```

Comment:
I'm tempted to remove some of these packages to make the list smaller. In particular anything called internal, and also the resources. Please try a test on OpenJDK to see if access is allowed to a classes in some internal package.